p03_6.py

```diff
@@ -0,0 +1,9 @@
+from p03_3 import testStatistic
+from p03_5 import subsample
+
+def nullDistribution( adData, iOne, iTwo ):
```

Comment:
No space after open parenthesis and before close parenthesis

Reply:
http://zh-google-styleguide.readthedocs.io/en/latest/google-python-styleguide/python_style_rules/#id7